pages/Documents/RichMessaging/WhatsAppBusinessMessagesTemplates/list-message-template.md

```diff
@@ -283,13 +279,11 @@ Footer of the list messages. This is shown below the subtitle or title if the fi
 },
 ```
 
-### Options Button
+### Options Menu Button
```

Comment:
Whole document: Please make the case consistent for all headings—some headings, like this one, use title case, others use sentence case.

@itay1313, do we already have a convention for this? It does look like title case but there are lots of inconsistencies.

Reply:
I'm now using title case everywhere for the time being.